build.py

```diff
@@ -18,7 +18,7 @@ def lint_faunadb():
 
 @task()
 def lint_tests():
-  test_disable = "no-member,no-self-use,protected-access,relative-import,too-many-public-methods"
+  test_disable = "bare-except,no-member,no-self-use,protected-access,relative-import,too-many-public-methods"
```

Comment:
With the switch to `assertRaises`, do we still need this? Bare excepts are generally not good practice, so i don't really like suppressing it everywhere.